source/manage/logging.rst

```diff
@@ -0,0 +1,487 @@
+Mattermost logging
+===================
+
+.. include:: ../_static/badges/allplans-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+.. contents:: On this page
+    :depth: 2
+
+By default, Mattermost writes logs to both the console and to the ``mattermost.log`` file in a machine-readable JSON format. 
+
+You can customize the following logging options based on your business practices and needs to by going to **System Console > Environment > Logging** or by editing the ``config.json`` file directly: 
+
+- `Omit incoming webhook details from logs </configure/environment-configuration-settings.html#enable-webhook-debugging>`__
```

Comment:
This link and the adjust link don't work, not sure if it's just an issue with the build or not.

Reply:
Actually, none of the absolute links seem to work. Build issue?

Reply:
Perfect!

Reply:
If the link is formatted as ``/dir/page.html`` rather than absolute as ``https://site.com``, it won't resolve as expected in the preview. It's a known issue.